glean-core/ffi/glean.h

```diff
@@ -359,6 +361,8 @@ char *glean_datetime_test_get_value_as_string(uint64_t metric_id, FfiStr storage
 
 uint8_t glean_datetime_test_has_value(uint64_t metric_id, FfiStr storage_name);
 
+char *glean_debug_view_tag(void);
```

Comment:
Maybe we should remove this (and the Swift test function that uses it)?  I don't think there's a real use case for *getting* it outside of Rust, and we can test that it's working by just looking at the HTTP headers.

Reply:
Removed this one and made it onl;y visible to the rust core crate. I also removed the`clear_debug_view_tag` function, which is not used anywhere. As much as it is wierd to me that add a `set` without a `remove`/`clear` there is no use case for it. If ever there is it is a simple enough function to just re-add.